Mathlib/CategoryTheory/Monoidal/Functor.lean

```diff
@@ -620,7 +620,7 @@ def comp : MonoidalFunctor.{v₁, v₃} C E :=
 #align category_theory.monoidal_functor.comp CategoryTheory.MonoidalFunctor.comp
 
 @[inherit_doc]
-infixr:80
+infixr : 80
```

Comment:
I don't recall seeing spaces with `infix` or `notation` more generally

Reply:
You're right; I meant to revert that change also.
```suggestion
infixr:80